cajita/src/Cajita_Halo.hpp

```diff
@@ -540,11 +541,12 @@ class Halo
                const Kokkos::View<int* [6], memory_space>& steering,
                const int element_idx,
                const std::integral_constant<std::size_t, 0>,
-               const ParameterPack<ArrayViews...>& array_views )
+               const Cabana::ParameterPack<ArrayViews...>& array_views )
```

Comment:
Note that this change of namespace for `ParameterPack` from `Cajita` to `Cabana` is a backwards compatibility break. We are already planning a bunch of small ones anyway elsewhere (e.g. in #342) so this doesn't really bother me that much.

Reply:
Added a wrapper to keep backwards compatibility